hydra-node/test/Hydra/Model.hs

```diff
@@ -378,19 +379,40 @@ runModel = RunModel{perform}
       StopTheWorld ->
         stopTheWorld
 
-performAbort :: MonadDelay m => Party -> StateT (Nodes m) m ()
+-- | Bring `Show` instance in scope drawing it from the `Action` type.
+-- This is a neat trick to provide `show`able results from action in a context where
+-- there's no explicit `Show a` instance, eg. in the `monitoring` and `postcondition`
+-- functions. We don't have access to an `a` directly because its value depends on
+-- type family `Realized`.
```

Comment:
Wat? What is this `Realized`? Other than undocumented https://hackage.haskell.org/package/quickcheck-dynamic-2.0.0/docs/Test-QuickCheck-StateModel.html#t:Realized